src/codelabs/implicit-animations.md

```diff
@@ -401,76 +419,66 @@ and watch how the animation changes when you pass the
              ElevatedButton(
 ```
 
-Now that you have passed `easeInOutBack` as the value for `curve` to
-`AnimatedContainer`, notice that the rates of change for `margin`,
-`borderRadius`, and `color` follow the curve defined by the
-`easeInOutBack` curve:
+When you pass the `Curves.easeInOutBack` constant to the `curve` property
+of the `AnimatedContainer` widget, watch how
+the rates of change for `margin`, `borderRadius`, and `color`
+follow the curve that constant defined.
```

Comment:
Right before the snippet, you already said "watch how". I think you can drop the "watch how" and switch to calling attention to what actually changed, similar to "notice" in the original text.